bakery/src/scripts/link_single.py

```diff
@@ -166,18 +173,31 @@ def transform_links(
                 f"from link {link}"
             )
         patch_link(node, source_book_uuid, canonical_book_uuid,
-                   canonical_book_slug, page_slug)
+                   canonical_book_slug, page_slug, target_book)
 
     save_linked_collection(output_path, doc)
 
 
 def main():
-    (baked_content_dir,
+    (
+        baked_content_dir,
         baked_meta_dir,
         source_book_slug,
-        output_path) = sys.argv[1:5]
+        output_path,
+    ) = sys.argv[1:5]
+
+    if len(sys.argv) > 5:
+        target_book = sys.argv[5]
+    else:
+        target_book = ""
```

Comment:
It may be worth adding a quick unit test for the invoking of `link-single` with a non-empty `target_book` string.